src/eko/evolution_operator/grid.py

```diff
@@ -135,7 +134,7 @@ def from_dict(
             config, q2_grid, thresholds_config, strong_coupling, interpol_dispatcher
         )
 
-    def get_threshold_operators(self, path):
+    def get_threshold_operators(self, path, is_backward):
```

Comment:
I believe this is too much information: `is_backward` could instead be attributed to `PathSegment` - this is the most atomic element (and perfectly aware)